otx/algorithms/common/adapters/mmcv/hooks.py

```diff
@@ -491,8 +491,8 @@ def _init_rule(self, rule, key_indicator):
         self.key_indicator = key_indicator
         self.compare_func = self.rule_map[self.rule]
 
-    def _should_check_stopping(self, runner):
-        """Called _should_check_stopping in ReduceLROnPlateauLrUpdaterHook."""
+    def _is_check_timing(self, runner):
+        """Called _is_check_timing in ReduceLROnPlateauLrUpdaterHook."""
         check_time = self.every_n_epochs if self.by_epoch else self.every_n_iters
         if not check_time(runner, self.interval):
```

Comment:
how about change this line to `return check_time(runner, self.interval)` and remove below lines?

Reply:
Reflected. Thank you for good idea!